infrastructure/zk/src/run/run.ts

```diff
@@ -10,16 +11,65 @@ import * as docker from '../docker';
 
 export { verifyKeys, dataRestore };
 
+type Token = {
+    decimals: number;
+    symbol: string;
+    name: string;
+    address: string;
+};
+
+type PartialFeeTickerConfig = {
+    fee_ticker: {
+        unconditionally_valid_tokens: string | string[];
+    };
+};
+
 export async function deployERC20(command: 'dev' | 'new', name?: string, symbol?: string, decimals?: string) {
     if (command == 'dev') {
-        await utils.spawn(`yarn --silent --cwd contracts deploy-erc20 add-multi '
-            [
-                { "name": "DAI",  "symbol": "DAI",  "decimals": 18 },
-                { "name": "wBTC", "symbol": "wBTC", "decimals":  8, "implementation": "RevertTransferERC20" },
-                { "name": "BAT",  "symbol": "BAT",  "decimals": 18 },
-                { "name": "GNT",  "symbol": "GNT",  "decimals": 18 },
-                { "name": "MLTT", "symbol": "MLTT", "decimals": 18 }
-            ]' > ./etc/tokens/localhost.json`);
+        await utils.spawn(`yarn --silent --cwd contracts deploy-erc20 add-multi '[
+            {
+              "decimals": 18,
+              "symbol": "RIF",
+              "name": "RSK Infrastructure Framework"
+            },
+            {
+              "decimals": 18,
+              "symbol": "rDOC",
+              "name": "rDOC"
+            },
+            { "name": "DAI",  "symbol": "DAI",  "decimals": 18 },
+            { "name": "wBTC", "symbol": "wBTC", "decimals":  8, "implementation": "RevertTransferERC20" },
+            { "name": "BAT",  "symbol": "BAT",  "decimals": 18 },
+            { "name": "GNT",  "symbol": "GNT",  "decimals": 18 },
+            { "name": "MLTT", "symbol": "MLTT", "decimals": 18 }
+          ]' > ./etc/tokens/localhost.json`);
+
+        // Update the fee ticker configuration file with the deployed unconditionally valid erc20 tokens
+        const localhostTokensFile = fs.readFileSync('./etc/tokens/localhost.json', 'utf-8');
+        const tokens: Token[] = localhostTokensFile && JSON.parse(localhostTokensFile);
```

Comment:
Why do we need to parse the tokens if we've just created the file with some known tokens?

Reply:
because we need to get the addresses, which are not known.

Reply:
I didn't realise that the `localhost.json` file contains the addresses